package.json

```diff
@@ -24,7 +24,8 @@
   "license": "Apache-2.0",
   "private": true,
   "devDependencies": {
-    "jshint": "latest"
+    "jshint": "latest",
+    "eslint": "2.9.0"
```

Comment:
Aren't we in the process of moving this to 4.x from 3.x?

Reply:
Apparently from 2.9.0 -> 4.16.0.  But I'll handle that/any conflicts from this.